### DISCUSSION TITLE
Customize UI and functions for individual rooms

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#5399] [Hubs Cloud Questions] [answered]
Hi，all，I want to modify some customized rooms on HUBS Cloud, such as:  
1. Customize the UI and buttons of the room without changing the whole hubs cloud  
2. Keep the object added by the user in the room, and you can see the content when you exit the room and re-enter it  
I want to know how to operate HUBS Cloud to realize these functions, looking forward to your reply!

**Comment by pattersonbl2** [comment, marked as answer]
This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun).  We have a #hubs-cloud channel that could help with this dev work.
